### PR TITLE
Update install-previous-sync-app.md

### PR DESCRIPTION
**Pull request by YuriySamorodov**
Split the step 5 to make it more clear

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/install-previous-sync-app.md [hunk 1]
```diff
@@ -67,21 +67,26 @@ The previous sync app (Groove.exe) is no longer installed by default with Office
 
     c. If the **User Account Control** dialog box appears, confirm that the action displayed is what you want, and then select **Continue**.
 
-5. At the command prompt, run the following commands, and press **Enter** after each one.
+5. At the command prompt, navigate to a folder you extracted Office Deployment tool into
```
Comment (JohanFreelancer9) [inline OneDrive/install-previous-sync-app.md:70]:
```suggestion
5. At the command prompt, run the following commands, and press **Enter** after each one:
```

File: OneDrive/install-previous-sync-app.md [hunk 2]
```diff
@@ -67,21 +67,26 @@ The previous sync app (Groove.exe) is no longer installed by default with Office
...
 
     ```DOS
-    cd /d %userprofile%\desktop setup.exe /configure AddODB.txt
+    cd /d %userprofile%\desktop
```
Comment (JohanFreelancer9) [inline OneDrive/install-previous-sync-app.md:73]:
```suggestion
    cd /d %userprofile%\desktop
    setup.exe /configure AddODB.txt
```

File: OneDrive/install-previous-sync-app.md [hunk 3]
```diff
@@ -67,21 +67,26 @@ The previous sync app (Groove.exe) is no longer installed by default with Office
...
+    ```
+6. Run command below
+
+    ```DOS
+    setup.exe /configure AddODB.txt
     ```
```
Comment (JohanFreelancer9) [inline OneDrive/install-previous-sync-app.md:79]:
```suggestion
```

File: OneDrive/install-previous-sync-app.md [hunk 4]
```diff
@@ -67,21 +67,26 @@ The previous sync app (Groove.exe) is no longer installed by default with Office
...
    
     > [!NOTE]
    > The installation occurs silently in the background and can take 20 minutes or more to complete, depending on the speed of your Internet connection. Don't close the Command Prompt window during the installation. When the installation is completed, a new command line appears at the command prompt.
 
-6. Select **Start**. In the **Start Search** box, type **OneDrive**, and then start OneDrive.
+7. Select **Start**. In the **Start Search** box, type **OneDrive**, and then start OneDrive.
```
Comment (JohanFreelancer9) [inline OneDrive/install-previous-sync-app.md:84]:
```suggestion
6. Select **Start**. In the **Start Search** box, type **OneDrive**, and then start OneDrive.
```

File: OneDrive/install-previous-sync-app.md [hunk 5]
```diff
@@ -67,21 +67,26 @@ The previous sync app (Groove.exe) is no longer installed by default with Office
...
 
     > [!NOTE]
     > After you set up OneDrive, you might not see sync icons on files until you restart your computer.
 
-7. Close the command prompt. You can now safely delete the following files from your desktop:
+8. Close the command prompt. You can now safely delete the following files from your desktop:
```
Comment (JohanFreelancer9) [inline OneDrive/install-previous-sync-app.md:89]:
```suggestion
7. Close the command prompt. You can now safely delete the following files from your desktop:
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - my suggestion.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**